apps/store/src/components/PriceCard/PriceCard.tsx

```diff
@@ -32,9 +35,15 @@ export const PriceCard = ({
         <Text size="m">{name}</Text>
       </CenteredText>
 
-      <PreviewText aria-disabled={cost === undefined}>
-        {currencyCode} {cost ?? '—'} /mth.
-      </PreviewText>
+      <PricePreviewContainer>
+        {!loading ? (
+          <PreviewText aria-disabled={cost === undefined}>
+            {currencyCode} {cost ?? '—'} /mth.
+          </PreviewText>
```

Comment:
nit: just a flavour thing to check for the truthy value first :) 
```suggestion
        {loading ? (
          <LoadingDots />
```